provider/go.mod

```diff
@@ -8,7 +8,9 @@ replace (
 )
 
 require (
-	github.com/hashicorp/terraform-plugin-sdk v1.9.1
+	github.com/hashicorp/terraform-plugin-sdk v1.9.1 // indirect
+	github.com/jfrog/terraform-provider-artifactory v0.0.0-20211108203830-a482006c47c5
 	github.com/pulumi/pulumi-terraform-bridge/v3 v3.9.0
-	github.com/pulumi/pulumi/sdk/v3 v3.14.0
+	github.com/pulumi/pulumi/sdk/v3 v3.14.1-0.20211007222624-789e39219452
```

Comment:
Should be `v3.17.0` (latest Pulumi version).

Reply:
Done, thanks to you :)